_includes/available-capacity-metric.md

```diff
@@ -0,0 +1 @@
+If you are running multiple nodes on a single machine (not recommended in production) and didn't specify the maximum allocated storage capacity for each node using the [`--store`](start-a-node.html#store) flag, the available capacity shown in the [**Capacity**](admin-ui-storage-dashboard.html#capacity) graph in the Admin UI is incorrect. This is because when multiple nodes are running on a single machine, the machine's hard disk is treated as an available store for each node, while in reality, only one hard disk is available for all nodes. The total available capacity is then calculated as the hard disk size multiplied by the number of nodes on the machine.
```

Comment:
We should probably expand this to also reference **Capacity Used** in the **Summary** panel:

```
... the **Capacity Used** metric in the [**Summary**](admin-ui-access-and-navigate.html#summary-panel) panel and the available capacity shown in the [**Capacity**](admin-ui-storage-dashboard.html#capacity) graph...
```

Reply:
Sweet, thanks!  Updated as suggested in bac307f9